google/cloud/spanner/benchmarks/single_row_throughput_plots.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Summarize the results from running throughput_vs_cpu_benchmark"""
+
+# %%
+import argparse
+import numpy as np
+import pandas as pd
+import plotnine as p9
+
+
+# %%
+def load_benchmark_output(file):
+    """Loads the output from the storage_throughput_vs_cpu_benchmark."""
```

Comment:
Nit: "storage_throughput_vs_cpu_benchmark" looks like a google-cloud-cpp remnant.  Similarly in the file comment above.

Reply:
Fixed in both places.